core/jazz_es-kinesis-log-streamer/test/test.js

```diff
@@ -0,0 +1,27 @@
+// =========================================================================
+// Copyright © 2017 T-Mobile USA, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+
+const assert = require('chai').assert;
+const index = require('../index');
+
+describe('Sample', function () {
+  it('tests handler', function (done) {
+
+    //Test cases to be added here.
```

Comment:
will need unit tests for this one

Reply:
Unit test will be included as part of follow up PR.